components/dashboard/src/Analytics.tsx

```diff
@@ -30,6 +30,12 @@ export interface TrackDotfileRepo {
     current: string;
 }
 
+export interface TrackFeedback {
+    score: number;
+    feedback: string;
```

Comment:
hi @laushinka!👋 i changed these two properties to match up with the property names of the `feedback_submitted` call from the website - i hope you don't mind me directly committing them to your baby!🙏🏼

Reply:
Of course, feel free! Thank you 🤗

Reply:
Will fix the text issue 🙏🏽

Reply:
Weird, I wonder why your commit got lost.. I'll make another one 🙏🏽